testsuite/MDAnalysisTests/transformations/test_fit.py

```diff
@@ -99,8 +99,11 @@ def test_fit_translation_no_masses(fit_universe):
     # create a universe without masses
     ref_u  = make_Universe()
     # what happens Universe without masses is given?
-    with pytest.raises(AttributeError): 
+    with pytest.raises(AttributeError):
+        fit_translation(test_u, ref_u, weights="mass")(ts)
```

Comment:
You have forgotten to remove your earlier test for an AttributeError, despite now expecting a TypeError.